src/DotNetty.Common/DotNetty.Common.csproj

```diff
@@ -9,7 +9,7 @@
     <AssemblyTitle>DotNetty: common routines</AssemblyTitle>
     <NeutralLanguage>en-US</NeutralLanguage>
     <VersionPrefix>0.7.3</VersionPrefix>
-	<VersionSuffix>preview1</VersionSuffix>
+	<VersionSuffix>neth</VersionSuffix>
```

Comment:
While technically `VersionSuffix` value doesn't make any difference, it is a part of semantic versioning and indicates the version is a pre-release one. With this said, it either should be something meaningful like `beta`, `preview`, etc. or you can omit it completely, so it will be a release version.